attributes/default.rb

```diff
@@ -59,20 +59,27 @@
 case node['platform_family']
 
 when 'rhel'
-  # RHEL5 edge case package set and portmap name
-  if node['platform_version'].to_i <= 5
-    default['nfs']['packages'] = %w(nfs-utils portmap)
-    default['nfs']['service']['portmap'] = 'portmap'
-  elsif node['platform_version'].to_i >= 7
-    default['nfs']['service']['lock'] = 'nfs-lock'
-    default['nfs']['service']['server'] = 'nfs-server'
-    default['nfs']['service']['idmap'] = 'nfs-idmap'
-
-    if node['platform_version'] == '7.0.1406'
-      default['nfs']['client-services'] = %w(nfs-lock.service)
+  case node['platform']
+    when 'amazon'
```

Comment:
The problem is that because the platform family for amazon is rhel and the version is based on the year it's always greater than 7, which means the attribute logic uses the rhel 7 values. Better to be correct for the latest versions of amazon than wrong for all of them IMO.